package.json

```diff
@@ -41,9 +41,11 @@
     "dependencies": {
         "@babel/polyfill": "7.0.0",
         "css-vars-ponyfill": "1.16.4",
-        "prop-types": "15.6.2",
-        "react": "16.8.6",
-        "react-dom": "16.8.6"
+        "prop-types": "^15.6.2"
```

Comment:
Prop-types is now listed in both in dependencies and devDependencies, is that intended?